docs/architecture/microservices/net-core-net-framework-containers/official-net-docker-images.md

```diff
@@ -42,8 +42,8 @@ When you explore the .NET image repositories at Docker Hub, you will find multip
 
 | Image | Comments |
 |-------|----------|
-| mcr.microsoft.com/dotnet/aspnet:**5.0** | ASP.NET Core, with runtime only and ASP.NET Core optimizations, on Linux and Windows (multi-arch) |
-| mcr.microsoft.com/dotnet/sdk:**5.0** | .NET 5, with SDKs included, on Linux and Windows (multi-arch) |
+| mcr.microsoft.com/dotnet/aspnet:**6.0** | ASP.NET Core, with runtime only and ASP.NET Core optimizations, on Linux and Windows (multi-arch) |
+| mcr.microsoft.com/dotnet/sdk:**6.0** | .NET 6, with SDKs included, on Linux and Windows (multi-arch) |
```

Comment:
Below this table, can you link to this page: https://github.com/dotnet/dotnet-docker ? Also, mention the avilability of Nightly builds too.

Reply:
sure, mentioned in after the table.